packages/framer-motion/src/animation/animators/js/index.ts

```diff
@@ -280,11 +280,13 @@ export function animateValue<V = number>({
             return millisecondsToSeconds(time)
         },
         set time(newTime: number) {
-            const timeInMs = secondsToMilliseconds(newTime)
+            newTime = secondsToMilliseconds(newTime)
```

Comment:
Reassigning rather than making new var as it was too easy to refer to the wrong one later down the func